### PR TITLE
Update Roadmap

### PR DESCRIPTION
**Pull request by mapedraza**
With completed tasks

### CODE REVIEW

File: doc/roadmap.md [hunk 1]
```diff
@@ -75,6 +74,7 @@ you wish to get involved in the implementation or influence the roadmap
 
 The following list contains all features that were in the roadmap and have already been implemented.
 
+- MQTT Retain flag ([#4388](https://github.com/telefonicaid/fiware-orion/issues/4388)) ([3.11.0](https://github.com/telefonicaid/fiware-orion/releases/tag/3.11.0))
```
Comment (mapedraza) [inline doc/roadmap.md:77]: 3.11.0 Pending to be released

Comment (mapedraza) [reply doc/roadmap.md:77]: > Should we hold this PR until 3.11.0 gets released?

I think it really does not matter, but we can wait till having the real tag

Comment (fgalan) [reply doc/roadmap.md:77]: Otherwise, the included link https://github.com/telefonicaid/fiware-orion/releases/tag/3.11.0 is not going to work...

Comment (fgalan) [reply doc/roadmap.md:77]: Orion 3.11.0 has ben released. This PR can now be merged.

**Pull request merged**